src/color/getColorContrast.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import getLuminance from './getLuminance'
+
+/**
+ * Returns the contrast ratio between two colors based on
+ * [W3's recommended equation for calculating contrast](http://www.w3.org/TR/WCAG20/#contrast-ratiodef).
+ *
+ * @example
+ * const contrastRatio = getColorContrast('#444', '#fff');
+ */
+export default function getColorContrast(
```

Comment:
Wondering if we should just call this `getContrast` to stay in line with `getLuminance`.